build.gradle

```diff
@@ -24,6 +24,8 @@ dependencies {
 	implementation 'org.mapstruct:mapstruct:1.5.5.Final'
 	annotationProcessor 'org.mapstruct:mapstruct-processor:1.5.5.Final'
 
+	testCompileOnly 'org.projectlombok:lombok:1.18.32'
```

Comment:
I think here should be `compileOnly` instead of `testCompileOnly`, same for the line below